radio/src/gui/128x64/model_outputs.cpp

```diff
@@ -143,7 +143,7 @@ void menuModelLimits(event_t event)
     lcdDrawChar(LIMITS_DIRECTION_POS, y, swVal);
 #endif
 
-    int8_t limit = (g_model.extendedLimits ? LIMIT_EXT_MAX : 100);
+    int limit = (g_model.extendedLimits ? LIMIT_EXT_MAX : 100);
```

Comment:
Yes it is indeed better for X7D, but an int is at least 16bits, this will increase flash usage on 8bits platforms!